ophyd/signal.py

```diff
@@ -1096,6 +1118,14 @@ def _write_pv_access(self, read_access, write_access, pv):
                        timestamp=self._metadata.get('timestamp'),
                        **self._metadata)
 
+    def destroy(self):
+        '''Destroy the EpicsSignal from the underlying PV instance'''
+        super().destroy()
+        if self._write_pv is not None:
+            self._write_pv.clear_auto_monitor()
```

Comment:
Maybe the control layer should provide a `release_pvs` method, symmetric to `get_pvs`. For the pyepics CL, it would call this. For the caproto CL, it would do the same + `pv._caproto_pv.go_idle()` which will disconnect the channel.